### PR TITLE
Existing VPC

### PR DESCRIPTION
**Pull request by sharepointoscar**
*Issue #, if available:*

*Description of changes:*
This version modifies the template to install within an existing VPC.

**NOTE:  T**he `00-master.yaml` file has various **TemplateURL** entries that need to be changed before this change works.  A future feature would allow to parametizing this, but if we need to deploy this soon.  Manually change the `00-master.yaml` file **TemplateURL** instances to match your existing AWS Bucket name and region.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: templates/00-master.yaml [hunk 1]
```diff
@@ -443,13 +392,13 @@ Parameters:
   WebAsgMax:
     AllowedPattern: ^((?!0$)[1-2]?[0-9]|30)$
     ConstraintDescription: Must be a number between 1 and 30.
-    Default: 1
+    Default: 2
```
Comment (rlorenzo) [inline templates/00-master.yaml:341]: @sharepointoscar Does this value need to be set to 2? Or is this for testing?

File: templates/04-web.yaml [hunk 2]
```diff
@@ -372,8 +372,8 @@ Resources:
   WebAutoScalingGroup:
     Type: AWS::AutoScaling::AutoScalingGroup
     Properties:
-      Cooldown: '60'
-      HealthCheckGracePeriod: 150
+      Cooldown: 60
+      HealthCheckGracePeriod: 3500
```
Comment (rlorenzo) [inline templates/04-web.yaml]: @sharepointoscar Why such a big increase from 150 to 3500?

File: templates/00-master.yaml (hunk 1 above)
Comment (sharepointoscar) [reply templates/00-master.yaml:341]: This is just for testing and you can change the value via the `.taskcat.yaml` which I provided with all parameters needed.

File: templates/04-web.yaml (hunk 2 above)
Comment (sharepointoscar) [reply templates/04-web.yaml]: Yep was testing, and set back to a lower value now.

**Comment by rlorenzo** [comment]
Closing in favor of https://github.com/ucla/aws-refarch-moodle/pull/7

**Pull request closed without merging**